ApiSamples/Base.lproj/Localizable.strings

```diff
@@ -166,6 +166,12 @@
 /* No comment provided by engineer. */
 "SampleDescription_PolygonWithElevation" = "Display a polygon on the map at a height above the ground";
 
+/* No comment provided by engineer. */
+"SampleTitle_AccuracyRingArroundBlueSphere" = "Blue sphere with accuracy ring arround";
```

Comment:
Spelling mistake in `arround`. We can replace the sentence with 
`Blue sphere with accuracy ring`